data_science/scripts/toggle_asg.py

```diff
@@ -3,63 +3,145 @@
 """
 Start/stop all the instances in an autoscaling group.
 
-Usage: toggle_asg.py (--start | --stop | --status) [--type=(p2 | t2)]
+Usage: toggle_asg.py (--start | --stop | --enable-overnight | --disable-overnight) [--type=(p2 | t2)]
+       toggle_asg.py [--type=(p2 | t2)]
```

Comment:
Consider adding `toggle_asg.py -h | --help` to this list, so you can get a usage message.

Currently it always does something, so getting to this message is quite hard!

Also, maybe `--start` et al should be non-prefix flags? So more like:

```
Usage: toggle_asg.py (start | stop | enable-overnight | disable-overnight | status) [--type=(p2 | t2)]
       toggle_asg.py -h | --help
```